ChallengesWithTestsMarkVII/ChallengesSet02.cs

```diff
@@ -1,58 +1,159 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace ChallengesWithTestsMarkVII
 {
     public class ChallengesSet02
     {
         public bool CharacterIsALetter(char c)
         {
-            throw new NotImplementedException();
+            if ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z'))
```

Comment:
This if-statement, and some of the others below, reads like "If true, return true. Otherwise return false."  Consider having just `return ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z'));`